tests/input_files/ref/sohex8_multiscale_macro/xxx-micro_model_1.pvd

```diff
@@ -0,0 +1,104 @@
+<?xml version="1.0"?>
+<!--
+# vtk DataFile Version 3.0
+-->
+<VTKFile type="Collection" version="0.1" ByteOrder="LittleEndian">
+  <Collection>
+    <DataSet timestep="0.00000000000000e+00" group="" part="0" file="xxx-vtk-files/micro_model_1-00000.pvtu"/>
+    <DataSet timestep="0.00000000000000e+00" group="" part="0" file="xxx-vtk-files/micro_model_1-00000.pvtu"/>
+    <DataSet timestep="0.00000000000000e+00" group="" part="0" file="xxx-vtk-files/micro_model_1-00000.pvtu"/>
+    <DataSet timestep="0.00000000000000e+00" group="" part="0" file="xxx-vtk-files/micro_model_1-00000.pvtu"/>
+    <DataSet timestep="0.00000000000000e+00" group="" part="0" file="xxx-vtk-files/micro_model_1-00000.pvtu"/>
+    <DataSet timestep="0.00000000000000e+00" group="" part="0" file="xxx-vtk-files/micro_model_1-00000.pvtu"/>
+    <DataSet timestep="0.00000000000000e+00" group="" part="0" file="xxx-vtk-files/micro_model_1-00000.pvtu"/>
+    <DataSet timestep="0.00000000000000e+00" group="" part="0" file="xxx-vtk-files/micro_model_1-00000.pvtu"/>
+    <DataSet timestep="0.00000000000000e+00" group="" part="0" file="xxx-vtk-files/micro_model_1-00000.pvtu"/>
+    <DataSet timestep="0.00000000000000e+00" group="" part="0" file="xxx-vtk-files/micro_model_1-00000.pvtu"/>
+    <DataSet timestep="0.00000000000000e+00" group="" part="0" file="xxx-vtk-files/micro_model_1-00000.pvtu"/>
+    <DataSet timestep="0.00000000000000e+00" group="" part="0" file="xxx-vtk-files/micro_model_1-00000.pvtu"/>
+    <DataSet timestep="0.00000000000000e+00" group="" part="0" file="xxx-vtk-files/micro_model_1-00000.pvtu"/>
+    <DataSet timestep="0.00000000000000e+00" group="" part="0" file="xxx-vtk-files/micro_model_1-00000.pvtu"/>
+    <DataSet timestep="0.00000000000000e+00" group="" part="0" file="xxx-vtk-files/micro_model_1-00000.pvtu"/>
+    <DataSet timestep="0.00000000000000e+00" group="" part="0" file="xxx-vtk-files/micro_model_1-00000.pvtu"/>
+    <DataSet timestep="0.00000000000000e+00" group="" part="0" file="xxx-vtk-files/micro_model_1-00000.pvtu"/>
+    <DataSet timestep="0.00000000000000e+00" group="" part="0" file="xxx-vtk-files/micro_model_1-00000.pvtu"/>
+    <DataSet timestep="0.00000000000000e+00" group="" part="0" file="xxx-vtk-files/micro_model_1-00000.pvtu"/>
+    <DataSet timestep="0.00000000000000e+00" group="" part="0" file="xxx-vtk-files/micro_model_1-00000.pvtu"/>
+    <DataSet timestep="0.00000000000000e+00" group="" part="0" file="xxx-vtk-files/micro_model_1-00000.pvtu"/>
+    <DataSet timestep="0.00000000000000e+00" group="" part="0" file="xxx-vtk-files/micro_model_1-00000.pvtu"/>
+    <DataSet timestep="0.00000000000000e+00" group="" part="0" file="xxx-vtk-files/micro_model_1-00000.pvtu"/>
+    <DataSet timestep="0.00000000000000e+00" group="" part="0" file="xxx-vtk-files/micro_model_1-00000.pvtu"/>
```

Comment:
This looks wrong to me. I think this will basically load `xxx-vtk-files/micro_model_1-00000.pvtu` 24 times in ParaView.  Why do you need to reference the same file here multiple times?